### ISSUE TITLE
problem on landmarks shift

### ISSUE THREAD

**Issue by PandaGuy0623** [geaxgx/depthai_blazepose#22]
Hi, I have this problem when I run your demo. The following picture shows that when I run the demo, the landmarks have a great shift from where they ought to be.
![blazepose](https://user-images.githubusercontent.com/103912082/181705129-2db5e19f-ce18-4381-a256-61e99a5ba151.png)
However, if I stream the image to the host and use mediapipe solution directly, the result is good.
![pose on host](https://user-images.githubusercontent.com/103912082/181705166-994708f1-3899-4fee-958d-bfe6cd3f9f1c.png)

I wonder if this problem is related to the blob file generated from tflite? How can I solve this problem?
 Thanks.

**Comment by geaxgx** [comment]
Yes, you are right, we have lost in accuracy when converting from tflite to blob. It happens for some models (similar problem with movenet). Here, one possible explanation is that we have less precision with float16 than in the original model (https://github.com/PINTO0309/tflite2tensorflow/issues/9#issuecomment-842460014). 
Another possible cause: https://github.com/openvinotoolkit/openvino/issues/7674
So there is not much we can do about it.
IIRC, the landmarks are less shifted with the 0.8.4 version of the model. You can try by using the older branch mediapipe084 (git checkout mediapipe084).
Otherwise, google should release (soon ?) a new version of their blazepose model, if I refer to this tweet: https://twitter.com/yeemachine/status/1539639359657447424

**Comment by PandaGuy0623** [comment]
Thanks for your reply!!

> Yes, you are right, we have lost in accuracy when converting from tflite to blob. It happens for some models (similar problem with movenet). Here, one possible explanation is that we have less precision with float16 than in the original model ([PINTO0309/tflite2tensorflow#9 (comment)](https://github.com/PINTO0309/tflite2tensorflow/issues/9#issuecomment-842460014)). Another possible cause: [openvinotoolkit/openvino#7674](https://github.com/openvinotoolkit/openvino/issues/7674) So there is not much we can do about it. IIRC, the landmarks are less shifted with the 0.8.4 version of the model. You can try by using the older branch mediapipe084 (git checkout mediapipe084). Otherwise, google should release (soon ?) a new version of their blazepose model, if I refer to this tweet: https://twitter.com/yeemachine/status/1539639359657447424

**Comment by tristanle22** [comment]
@PandaGuy0623 Have you been able to get around this issue?
